controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -331,7 +331,6 @@ func (r *StorageClusterReconciler) deleteResources(sc *ocsv1.StorageCluster) (re
 		&ocsCephFilesystems{},
 		&ocsCephBlockPools{},
 		&ocsSnapshotClass{},
-		&ocsGroupSnapshotClass{},
```

Comment:
its removed because we dont support VGSClass in external mode right?

Reply:
Yes, right